src/datalink/bpf.rs

```diff
@@ -27,11 +26,12 @@ use util::NetworkInterface;
 
 // NOTE buffer must be word aligned.
 #[inline]
-pub fn datalink_channel(network_interface: &NetworkInterface,
-                        write_buffer_size: usize,
-                        read_buffer_size: usize,
-                        channel_type: DataLinkChannelType)
-    -> io::Result<(Box<EthernetDataLinkSender>, Box<EthernetDataLinkReceiver>)> {
+pub fn datalink_channel
+    (network_interface: &NetworkInterface,
```

Comment:
Yes, it still destroys that special format. I'm divided here. Configuring the formatter brings back the human factor :( It would be so beautiful to remove that completely.
It's possible to add `#[cfg_attr(rustfmt, rustfmt_skip)]` to make it skip formating, but I only get that to work for entire files, not individual lines or functions.

I'll simply back out the reformatting.